docsk8s/quick-start/sample-configuration-files.asciidoc

```diff
@@ -33,62 +45,243 @@ This contains the Logstash settings and pipeline configuration:
+              memory: 2Gi
+            requests:
+              cpu: 1000m
+              memory: 2Gi
+          ports: <4>
+            - containerPort: 9600
+              name: stats
+            - containerPort: 5044
+              name: beats
+          livenessProbe: <5>
+            httpGet:
+              path: /
+              port: 9600
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            timeoutSeconds: 5
+            failureThreshold: 3
+          readinessProbe: <6>
+            httpGet:
+              path: /
+              port: 9600
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            timeoutSeconds: 5
+            failureThreshold: 3
+          volumeMounts: <7>
+            - name: logstash-pipeline
+              mountPath: /usr/share/logstash/pipeline
+            - name: logstash-config <8>
+              mountPath: /usr/share/logstash/config/logstash.yml
+              subPath: logstash.yml
+            - name: es-certs <9>
+              mountPath: /usr/share/logstash/config/es_ca.crt
+              subPath: ca.crt
+            - name: logstash-beats-tls
+              mountPath: /usr/share/logstash/config/ca.crt
+              subPath: ca.crt
+            - name: logstash-beats-tls
+              mountPath: /usr/share/logstash/config/server.pkcs8.key
+              subPath: server.pkcs8.key
+            - name: logstash-beats-tls
+              mountPath: /usr/share/logstash/config/server.crt
+              subPath: server.crt
+      volumes:
+        - name: logstash-pipeline <7>
+          configMap:
+            name: logstash-pipeline
+        - name: logstash-config <8>
+          configMap:
+            name: logstash-config
+        - name: es-certs <9>
+          secret:
+            secretName: demo-es-http-certs-public
+        - name: logstash-beats-tls <10>
+          secret:
+            secretName: logstash-beats-tls
+        - name: es-user <11>
+          secret:
+            secretName: demo-es-elastic-user
 --
-+
-* `logstash-beats-tls` is the secret containing the `ca.crt`, `server.crt` and `server.pkcs8.key` to input data from Filebeat and Metricbeat.
-* `demo-es-http-certs-public` contains the CA certificate to output data to {es}. Refer to link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-tls-certificates.html[TLS certificates] in the {eck} Guide for details.
-* The {es} password is taken from `demo-es-elastic-user` and passed to the Logstash pipeline as an `ELASTICSEARCH_PASSWORD` environment variable. Refer to link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-request-elasticsearch-endpoint.html[Access the {es} endpoint] in the {eck} Guide for details.
 
+<1> Logstash docker image
+<2> If you want to change any JVM settings from the default - most commonly setting memory. allocations - prefer to set them in the `LS_JAVA_OPTS` environment variable than recreating the whole `jvm.options` file in a `ConfigMap`.
```

Comment:
I wonder if we reframing this as a general guide to setting up logstash, using the configuration files as context, rather than as the subject might make more sense here. And then we can split up in a more natural way rather than per file?